src/renderer/shared/ui-entities/Hash/Truncate.tsx

```diff
@@ -27,16 +27,21 @@ export const Truncate = memo(({ text, ellipsis = '...' }: Props) => {
 
   const [gotFirstCalculation, setGotFirstCalculation] = useState(false);
   const [truncatedText, setTruncatedText] = useState(text);
+  const [measurements, setMeasurements] = useState<{ container: number; ellipsis: number; text: number }>({
+    container: 0,
+    ellipsis: 0,
+    text: 0,
+  });
```

Comment:
Can rely on inferred types
```suggestion
  const [measurements, setMeasurements] = useState({ container: 0, ellipsis: 0, text: 0 });
```